src/app/Plans/components/PlanStatus.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { IMigration, IPlan } from '@app/queries/types';
+import { StatusIcon, StatusType } from '@konveyor/lib-ui';
+import { Level, LevelItem, Progress, ProgressMeasureLocation } from '@patternfly/react-core';
+
+interface IPlanStatusProps {
+  plan: IPlan;
+  migration: IMigration;
+}
+
+const PlanStatus: React.FunctionComponent<IPlanStatusProps> = ({
+  plan,
+  migration,
+}: IPlanStatusProps) => {
+  if (plan.status.conditions.every((condition) => condition.type === 'Ready')) {
+    return <StatusIcon status={StatusType.Ok} label="Ready" />;
+  } else {
+    // TODO: This is only a placeholder, more statuses check needs to be done.
+    const totalVMs = plan.spec.vmList.length;
+    const percentVMsDone = totalVMs > 0 ? (migration.status.nbVMsDone * 100) / totalVMs : 0;
+    const label = 'Running';
+
+    return (
+      <>
+        <Level>
+          <LevelItem>
```

Comment:
@gildub hey, sorry I steered you in the wrong direction here. There is still some weird spacing around this `<Level>`, I checked the docs for the Progress component and realized there are props to display this stuff for us. We can use the `label` prop for the status icon and the `valueText` prop for "VMs migrated" text, like in this example: https://www.patternfly.org/v4/components/progress/#finite-step

Reply:
@mturley, I actually initially used Progress props but there is an issue with title prop which doesn't like passing it a react element. I was thinking about creating an issue on PF project.

Reply:
Ah, ok, I mixed up the `title` and `label` props. Ok, yeah, we can just put a `className={spacing.mbXs}` on the Level I guess to give it some bottom margin. Doing this though I notice there is still some weird white space to the right of the progress bar / the label hangs over the edge of it. Not sure what to do about that.

Yes, let's open an issue on patternfly-react for it, that `title` prop should be a `React.ReactNode` instead of a `string`.

Reply:
It looks like we can solve the weird overhang by also putting `spacing.prMd` on the Level as well, at least until we address the upstream PF issue. So this should make it look right:

```ts
import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
...
className={`${spacing.mbXs} ${spacing.prMd}`}
```
```

Reply:
Thanks @mturley, I've created https://github.com/patternfly/patternfly-react/issues/4875
Using the spacing work arounds in the meantime.

Reply:
Perfect, thanks for opening that!